docs/usage/guides/exposing_services.md

```diff
@@ -32,7 +32,7 @@ Therefore, we have to create the cluster in a way, that the internal port 80 (wh
   *Note*: `k3s` deploys [`traefik`](https://github.com/containous/traefik) as the default ingress controller
 
     ```YAML
-    apiVersion: extensions/v1beta1
+    apiVersion: networking.k8s.io/v1
```

Comment:
Maybe we should just add a note here, that this is starting from Kubernetes v1.19

Reply:
It is already deprecated and will be removed with Kubernetes 1.22. This should not be an problem, since k3d comes with Kubernetes 1.19 by default.

Reply:
That's true for sure, but in between one had to use `networking.k8s.io/v1beta1` (I think starting with v1.16), before it went GA in v1.19.
Not that big of a problem though.